var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -1040,14 +1057,15 @@ def cmake_args(self):
             self.define_from_variant("CP2K_USE_PLUMED", "plumed"),
             self.define_from_variant("CP2K_USE_SPGLIB", "spglib"),
             self.define_from_variant("CP2K_USE_VORI", "libvori"),
-            self.define_from_variant("CP2K_USE_SPLA", "spla"),
```

Comment:
Is this correct? As far as I can tell, `CP2K_USE_SPLA_GEMM_OFFLOADING` is a dependent option of `CP2K_USE_SPLA`.

Reply:
We should change the logic. It is confusing. Why not suppressing SPLA_OFFLOADING variant completely and keep the `spla` which will imply `spla_offloading`. it is the only case where cp2k uses spla so it should not be constraining to only consider `spla_offloading` automatically. Thoughts ?

Reply:
If it is the only case where CP2K uses SPLA that is fine.  But then the CMake logic is also off, and needs to be fixed IMO. 

Since SPLA is also available without GPU support, I thought CP2K could also use it without GPU support (and this is useful to allow, for debugging or development purposes for example).